src/main/resources/assets/js/app/wizard/action/ContentWizardActions.ts

```diff
@@ -471,6 +491,31 @@ export class ContentWizardActions
         return true;
     }
 
+    private updateResetActionState() {
+        this.isResetToBeVisible().then((value: boolean) => this.actionsMap.RESET.setVisible(value)).catch(DefaultErrorHandler.handle);
+    }
+
+    private isResetToBeVisible(): Q.Promise<boolean> {
+        const parentProjectName: string = ProjectContext.get().getProject().getParent();
+
+        if (!parentProjectName) {
+            return Q(false);
+        }
+
+        const id: string = this.content.getId();
+
+        return new ContentsExistRequest([id])
+            .setRequestProjectName(parentProjectName)
+            .sendAndParse()
+            .then((result: ContentsExistResult) => {
+                if (!!result.getContentsExistMap()[id]) {
+                    return (this.content.getInherit().length * 2) < Object.keys(ContentInheritType).length;
```

Comment:
Let's implement `isFullyInherited` in `ContentSummaryAndCompareStatus` and have all the checks there.